docs/adv-topics/usbinterfaces.md

```diff
@@ -9,6 +9,36 @@ Setting up USB audio interfaces is much easier with ASL3.
+
+Most nodes use the SimpleUSB channel driver. Nodes that require more advanced features such as DSP squelch detection, CTCSS tone encode/decode or additional audio filtering options will use the USBRadio channel driver. These channel drivers both have tune utilities for setting audio levels and other functions. Once a node has been configured in the `asl-menu` "AllStar Node Setup Menu", select the "Interface Tune CLI" menu option. The following options should then be used to check and optimize audio level settings:
+
+**Note: Be sure you are not connected to any other nodes when doing any of the following tests and adjustments.**
+
+ - `Set Rx Voice Level (using display)` provides a level meter. Talk at a normal and steady volume into the microphone and be sure that the average level does not go past the "3KHz" point, and that peak levels do not go significantly past the "5KHz" point. Note that for proper mic technique you should talk no closer than 2 to 3 inches (5 - 10 cm) from the microphone, and it should be held at an angle to minimize pops/plosives. Once you see where the audio levels are hit the Enter key and you'll then be prompted to change the setting. If for example the current setting is 500 and your levels were too high, try entering 400 for a new value and then repeat the test. Continue to iteratively adjust the settings until the average audio levels stay around the "3KHz" mark. For a more precise calibration with radio nodes a 1KHz test tone can be input into another radio at a level that results in 3KHz FM deviation, then calibrate the node to the 3KHz mark, however this can be more complicated to set up and may not account for other differences such as different mics or mic gain settings.
+ - `Set Transmit A Level`, `Set Transmit B Level` (or `Set Transmit Voice Level` for USBRadio) allow the node audio output levels to be set. For nodes connected to a radio transceiver the level should be set such that the transmitted FM deviation is no higher than 3KHz and the audio does not sound distorted or overly loud. When listening on another radio it can also be helpful to confirm the audio levels from your node sound about the same as levels on FM repeaters in your area.
+ - `Toggle RX Boost` enables a preamp in the CM1xx for use with very low mic-level input signals, This should be disabled in almost all cases.
+ - `View Rx Audio Statistics` shows precise audio level information. If when running this option and talking at a normal to loud volume into your mic you see `Pk` levels of any higher than -3.0 dBFS, `Avg Pwr` levels of higher than -12 dBFS, or any `ClipCnt` value greater than 0, you should reduce your rxmix setting (using the `Set Rx Voice Level` menu option) and then repeat this step.
```

Comment:
I am wondering if it would be helpful to explain the term dBFS?

Reply:
Agreed, I added some text there defining dBFS.